test/communication/EventFeed.AspNet.Tests.Integration/PageMetaTests.cs

```diff
@@ -33,7 +35,7 @@ public async Task With_no_events_pages_count_is_1()
             using var host = await A.Host.Build();
 
             var response = await host.GetTestClient().GetAsync("/api/event-feed");
-            var pageMeta = JsonSerializer.Deserialize<PageMeta>(await response.Content.ReadAsStringAsync());
+            var pageMeta = JsonSerializer.Deserialize<PageMeta>(await response.Content.ReadAsStringAsync(), _jsonSerializerOptions);
```

Comment:
Same comment as middleware. Can we pull the serialize method out. It is a detail that shouldn't be important for most tests. 
From a "tests as documentation" point of view, I would like some test that shows the serialization options are now necessary for correct functioning of the serialization. It fails to deserialize properly without the options?